lib/domain.js

```diff
@@ -132,10 +116,10 @@ internalBinding('domain').enable(topLevelDomainCallback);
 
 function updateExceptionCapture() {
   if (stack.every((domain) => domain.listenerCount('error') === 0)) {
-    setUncaughtExceptionCaptureCallback(null);
+    setUncaughtExceptionCaptureCallback(CAPTURE_CB_KEY, null);
   } else {
-    setUncaughtExceptionCaptureCallback(null);
-    setUncaughtExceptionCaptureCallback((er) => {
+    setUncaughtExceptionCaptureCallback(CAPTURE_CB_KEY, null);
```

Comment:
Setting this to `null` before setting it to an actual callback is actually not needed anymore, I'll update this shortly.